<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChiyoControls</class>
 <widget class="QWidget" name="ChiyoControls">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="baseSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="ChiyoGraphics" name="graphicsView"/>
   </item>
   <item>
    <widget class="QToolBar" name="controlToolBar">
     <property name="iconSize">
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
     <addaction name="action_zoom_in"/>
     <addaction name="action_zoom_out"/>
     <addaction name="action_zoom_original"/>
     <addaction name="action_zoom_fit"/>
     <addaction name="separator"/>
     <addaction name="action_scale"/>
     <addaction name="separator"/>
    </widget>
   </item>
  </layout>
  <action name="action_zoom_in">
   <property name="icon">
    <iconset resource="ChiyoGUI.qrc">
     <normaloff>:/Resources/icons/gtk-zoom-in.png</normaloff>:/Resources/icons/gtk-zoom-in.png</iconset>
   </property>
   <property name="text">
    <string>Zoom &amp;In</string>
   </property>
   <property name="toolTip">
    <string>Zoom in the image.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+=</string>
   </property>
  </action>
  <action name="action_zoom_out">
   <property name="icon">
    <iconset resource="ChiyoGUI.qrc">
     <normaloff>:/Resources/icons/gtk-zoom-out.png</normaloff>:/Resources/icons/gtk-zoom-out.png</iconset>
   </property>
   <property name="text">
    <string>Zoom &amp;Out</string>
   </property>
   <property name="toolTip">
    <string>Zoom out the image.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+-</string>
   </property>
  </action>
  <action name="action_zoom_original">
   <property name="icon">
    <iconset resource="ChiyoGUI.qrc">
     <normaloff>:/Resources/icons/gtk-zoom-100.png</normaloff>:/Resources/icons/gtk-zoom-100.png</iconset>
   </property>
   <property name="text">
    <string>Zoom Original</string>
   </property>
   <property name="toolTip">
    <string>Zoom the image to origin.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+1</string>
   </property>
  </action>
  <action name="action_zoom_fit">
   <property name="icon">
    <iconset resource="ChiyoGUI.qrc">
     <normaloff>:/Resources/icons/gtk-zoom-fit.png</normaloff>:/Resources/icons/gtk-zoom-fit.png</iconset>
   </property>
   <property name="text">
    <string>Zoom &amp;Fit</string>
   </property>
   <property name="toolTip">
    <string>Zoom the image to a fit scale.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+0</string>
   </property>
  </action>
  <action name="action_scale">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>100%</string>
   </property>
   <property name="toolTip">
    <string>Current scale</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ChiyoGraphics</class>
   <extends>QGraphicsView</extends>
   <header location="global">chiyographics.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="ChiyoGUI.qrc"/>
 </resources>
 <connections/>
</ui>
